chart/templates/server-ide-configmap.yaml

```diff
@@ -104,6 +104,34 @@ options:
 
 defaultIde: "code"
 defaultDesktopIde: "code-desktop"
+
+clients:
+  vscode:
+    defaultDesktopIDE: "code-desktop"
+    desktopIDEs: ["code-desktop"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
+      "If you don't see the dialog, make sure you have <a target='_blank' class='gp-link' href='https://code.visualstudio.com/download'>VS Code</a> installed on your machine.",
+      "After that, click the <b>${OPEN_LINK_LABEL}</b> button below."
```

Comment:
**suggestion**: I tried simplifying the instructions here. Let me know if you think it would be better to keep instructions into separate senteces, etc. See `BEFORE` / `AFTER` screenshots above in the review comment.